velox/functions/lib/tests/DateTimeFormatterTest.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/lib/DateTimeFormatter.h"
+#include "velox/common/base/Exceptions.h"
+
+#include <gtest/gtest.h>
+
+using namespace facebook::velox;
+
+namespace facebook::velox::functions {
+
+class DateTimeFormatterTest : public testing::Test {};
+
+TEST_F(DateTimeFormatterTest, builder) {
```

Comment:
can we split this into smaller test cases? Ideally you want something that could give you a reasonable guess about what failed as the test case name.